### PR TITLE
PLAT-260: functests on CI

### PR DESCRIPTION
**Pull request by ita-sammann**
Added functests step to `common.yaml` workflow

### CODE REVIEW

File: ledger-core/v2/Makefile [hunk 1]
```diff
@@ -51,6 +51,7 @@ clean: ## run all cleanup tasks
 	go clean $(ALL_PACKAGES)
 	rm -f $(COVERPROFILE)
 	rm -rf $(BIN_DIR)
+	rm -rf $(ARTIFACTS_DIR)
```
Comment (ElSacramento) [inline ledger-core/v2/Makefile:54]: I can't find using `make clean` in github actions.

Comment (ita-sammann) [reply ledger-core/v2/Makefile:54]: It's used in `launchnet.sh`

Comment (ElSacramento) [reply ledger-core/v2/Makefile:54]: In `rebuild_binaries`? but func is unused :(

Comment (ita-sammann) [reply ledger-core/v2/Makefile:54]: Hmm, yes, really. But nevertheless, I use it all the time manually :)

Comment (ElSacramento) [reply ledger-core/v2/Makefile:54]: Maybe it's time to add it in `test_func` step or in `launchnet` :)

Comment (ita-sammann) [reply ledger-core/v2/Makefile:54]: To run tests on a clean environment I run `make` (there is `clean` target in it) and then `make test_func`

**Pull request merged**